src/build.d

```diff
@@ -1467,16 +1479,127 @@ class Dependency
             scope (failure) if (!verbose) dump();
 
             if (commandFunction !is null)
+            {
+                scope (exit) writefln!"%s: Finished %s.commandFunction"(taskList, what);
 
                 return commandFunction();
-
+            }
             if (command)
             {
+                scope (exit) writefln!"%s: Finished %s - %-(%s %)"(taskList, what, command[0 .. min(6, $)]);
                 command.run;
             }
         }
     }
 
+     /// Metadata required for parallel execution
+    private
+    {
+        alias Task = typeof(task(&Dependency.init.runRecursive));
```

Comment:
one private per field, I think that's the convention and it make it easier to maintain since you don't have to poke around the file to search for all the attributes that could apply to any field.

Reply:
I think this is fine as it's just a short block and marked by curly braces. It's just the `private:` labels which are a bit tricky to spot.

Reply:
The intention here was to emphasize that these attributes are related (as given by the leading comment). But i can change it as well.